ctms/schemas/pending_acoustic_record.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import Field
+
+from .base import ComparableBase
+
+
+class PendingAcousticRecordBase(ComparableBase):
+    """Data that is included in input/output/db of a pending_acoustic_record and such."""
+
+    retry: int = Field(
+        default=0,
+        description="Set to the retry attempt.",
+    )
+    created_date: Optional[str] = Field(
+        default=None,
+        description="ISO 8601 Timestamp with utc",
+        example="2021-01-29T18:43:49.082375+00:00",
+    )
+
+    update_timestamp: datetime = Field(
+        default_factory=datetime.utcnow,
```

Comment:
[datetime.utcnow](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) returns the current UTC time as a naive datetime, which is often interpreted as a local time. The Python docs recommend using [datetime.now(timezone.utc)](https://docs.python.org/3/library/datetime.html#datetime.datetime.now) instead. This bit me about a month ago, fixed in PR #113.

I think this works (you'll need to import ``timezone`` from ``datetime`` as well):
```suggestion
        default_factory=lambda: datetime.now(timezone.utc),
```

Reply:
Ah, totally makes sense, thanks! I wasn't sure how to opt for the callable, the lambda is perfect!